packages/account/src/providers/operations.graphql

```diff
@@ -6,6 +6,42 @@
 # Fragments
 
 fragment transactionStatusFragment on TransactionStatus {
+  type: __typename
+  ... on SubmittedStatus {
+    time
+  }
+  ... on SuccessStatus {
+    block {
+      id
+    }
+    time
+    receipts {
+      ...receiptFragment
+    }
+    programState {
+      returnType
+      data
+    }
+    receipts {
+      ...receiptFragment
+    }
+  }
+  ... on FailureStatus {
+    block {
+      id
+    }
+    time
+    reason
+    receipts {
+      ...receiptFragment
+    }
+  }
+  ... on SqueezedOutStatus {
+    reason
+  }
+}
+
+fragment transactionStatusSubscriptionFragment on TransactionStatus {
```

Comment:
Tooling doesn't allow me to make a suggestion here, but you can simplify this fragment:
```gql
fragment transactionStatusSubscriptionFragment on TransactionStatus {
  type: __typename
  ... on SqueezedOutStatus {
    reason
  }
}
```
The `type` and `SqueezedOutStatus.reason` fields are the only ones used so we can reduce bandwidth by removing the other ones. I tried it locally and tests passed.

Reply:
https://github.com/FuelLabs/fuels-ts/pull/2021/commits/a3e9c6e369d883bf311580014413deedbb757626